lib/modules/hosts/iloopit.js

```diff
@@ -3,32 +3,26 @@ addLibrary('mediaHosts', 'iloopit', {
 	domains: ['iloopit.net'],
 	logo: '//iloopit.net/favicon.ico',
 	detect: href => (
-		(/^https?:\/\/(?:\w+\.)?iloopit\.net\/tube\/(\d+)/i).exec(href) ||
-		(/^https?:\/\/(?:\w+\.)?iloopit\.net\/[\-a-z\ \/%20]+?\/\?type=looplayer&loopid=(\d+)/i).exec(href) ||
-		(/^https?:\/\/(?:\w+\.)?iloopit\.net\/(\d+)\/[a-z\-\/]+((\?type=looplayer)|(\?type=embed))?/i).exec(href)
+		(/^https?:\/\/(?:\w+\.)?iloopit\.net\/.+?\/\?type=looplayer&loopid=(\d+)/i).exec(href) ||
+		(/^https?:\/\/(?:\w+\.)?iloopit\.net(?:\/tube\/)?\/(\d+)\/.+?\/(?:(?:\?type=looplayer)|(?:\?type=embed))?/i).exec(href)
 	),
 	async handleLink(elem, [, hash]) {
```

Comment:
`async` should be removed since nothing is `await`ed in this function.
And `hash` should be removed since you're not using it (that will fix the build).